sdk/src/main/java/com/sygic/travel/sdk/di/favoritesModule.kt

```diff
@@ -0,0 +1,19 @@
+package com.sygic.travel.sdk.di
+
+import android.arch.persistence.room.Room
+import com.github.salomonbrys.kodein.Kodein
+import com.github.salomonbrys.kodein.bind
+import com.github.salomonbrys.kodein.instance
+import com.github.salomonbrys.kodein.singleton
+import com.sygic.travel.sdk.favorites.db.StDb
+import com.sygic.travel.sdk.favorites.facade.FavouritesFacade
+import com.sygic.travel.sdk.favorites.service.FavoriteService
+
+
+internal val favoritesModule = Kodein.Module {
+	bind<FavoriteService>() with singleton {
+		FavoriteService(Room.databaseBuilder(instance(), StDb::class.java, "st-sdk-db").build())
```

Comment:
please add type to instance call, it's not obvious.

Reply:
:heavy_check_mark: